src/back/spv/writer.rs

```diff
@@ -1375,7 +1450,20 @@ impl Writer {
                 }
             }
             crate::Expression::GlobalVariable(handle) => self.global_variables[handle.index()].id,
-            crate::Expression::Constant(handle) => self.constant_ids[handle.index()],
+            crate::Expression::Constant(handle) => {
```

Comment:
in the near future, when we know how each constant is used by each function - #656 - we'll be able to do this at the beginning of the function only. Maybe we can leave a small TODO pointing to this issue?